mne/forward/forward.py

```diff
@@ -180,11 +180,11 @@ def _get_src_type_and_ori_for_repr(self):
         src_types = np.array([src["type"] for src in self["src"]])
 
         if (src_types == "surf").all():
-            src_type = "Surface with {self['nsource']} vertices"
+            src_type = f"Surface with {self['nsource']} vertices"
```

Comment:
FYI snuck this in there because I came across it today and isn't worth a separate PR